src/utils/setAuthToken.ts

```diff
@@ -0,0 +1,6 @@
+import axios from 'axios';
+
+export const setAuthToken = (token: string) => {
+    localStorage.setItem('token', token);
```

Comment:
The same situation like with getting token. But here, you can accurate say what this line is doing, e.i. you can say "setTokenIntoLocalStorage(token)" by using method like this. It sound weird at first, but remember that you wrote less code than you read the code